v2/pkg/reporting/format/format.go

```diff
@@ -154,3 +168,23 @@ func GetMatchedTemplate(event *output.ResultEvent) string {
 	template := builder.String()
 	return template
 }
+
+func ToMarkdownTableString(templateInfo *model.Info) string {
+	fields := map[string]string{
```

Comment:
It's not certain that all these fields will be provided in a template, at least description may not be present everywhere, so we should handle those scenarios as well as it will mess up the output in case of no values. Same goes for Jira formatting logic.

Reply:
Fixed in df3090c0b412599c0c6187c9be5d741b8ee450c1